specs/MultiProfile.md

```diff
@@ -118,6 +124,49 @@ HRESULT AppWindow::OnCreateCoreWebView2ControllerCompleted(HRESULT result, ICore
     // ...
 }
 ```
+
+### Access and use cookie manager from profile
+
+```cpp
+wil::com_ptr<ICoreWebView2CookieManager> m_cookieManager;
+ScenarioCookieManagement::ScenarioCookieManagement(ICoreWebView2Controller* controller)
+{
+    wil::com_ptr<ICoreWebView2> coreWebView2;
+    CHECK_FAILURE(controller->get_CoreWebView2(&coreWebView2));
+    auto webview7 = coreWebView2.try_query<ICoreWebView2_7>();
+    if (webview7)
+    {
+        wil::com_ptr<ICoreWebView2Profile> profile;
+        CHECK_FAILURE(webview7->get_Profile(&profile));
+        auto profile2 = profile.try_query<ICoreWebView2Profile2>;
+        if (profile2){
+        //! [CookieManagerProfile]
```

Comment:
You can remove the //! comments from this API spec. But for the actual sample code, just one line of calling get_CookieManager is not very helpful. You can put the //! around this whole ScenarioCookieManagement::ScenarioCookieManagement function.

Reply:
```suggestion
```